tests/integration/domain/health.spec.ts

```
@@ -0,0 +1,37 @@
+import { test, expect } from '../../fixtures/base-fixtures'
+import { expectValidResponse } from '../../utils/assertions'
+
+test.describe('Domain Health Endpoint', () => {
+  test('should return healthy status', async ({ }) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)